README.md

```diff
@@ -29,7 +29,14 @@ Install from pip:
 pip install aga
 ```
 
-or with the python dependency manager of your choice (I like [poetry](https://github.com/python-poetry/poetry)).
+or with the python dependency manager of your choice (I like [poetry](https://github.com/python-poetry/poetry)), for example:
+
+```bash
+pip install poetry
```

Comment:
Can we put the [official](https://python-poetry.org/docs/#installation) installation method? Just change this line to

```bash
curl -sSL https://install.python-poetry.org | python3 -
```

Reply:
(Installing python libraries globally via `pip` is almost always a bad idea.)

Reply:
I have tried to fix my PR with your suggestion, but I'm getting mixed messages from github. Fingers crossed.